### PR TITLE
feat: tooltip element

### PR DESCRIPTION
**Pull request by georgeciubotaru**

### CODE REVIEW

File: src/components/Heading/template.pug [hunk 1]
```diff
@@ -1,21 +1,21 @@
 mixin content
     +if('level === 1')
-        h1(class="heading {hoverableClass} {displayClass} {asLink ? '' : className}")
+        h1(use:forwardAction bind:this="{forwardRef}" class="heading {hoverableClass} {displayClass} {asLink ? '' : className}")
```
Comment (zolotokrylin) [inline src/components/Heading/template.pug:3]: @georgeciubotaru @markholdex what's the use case when we are using Headings with tooltip 🤔 ?

**Review by zolotokrylin — COMMENTED** [review]
@georgeciubotaru where can we learn (as users of the tool tip) how it should be used in the code? from dev perspective. Like an intro to how to use

File: src/components/Heading/template.pug (hunk 1 above)
Comment (georgeciubotaru) [reply src/components/Heading/template.pug:3]: @zolotokrylin we don't have now any use-case. That's for future use

**Comment by georgeciubotaru** [comment]
@zolotokrylin please take a look here
https://60dc698bd6a4f60039a51ae4-ojeilfyrzm.chromatic.com/?path=/docs/example-tooltip--default-with-html

**Review by markholdex — APPROVED** [review]

**Pull request merged**